integration-tests/http/__tests__/price-preference/admin/price-preference.spec.ts

```diff
@@ -0,0 +1,223 @@
+          {
+            attribute: "region_id",
+            value: "region-1",
+            is_tax_inclusive: true,
+          },
+          adminHeaders
+        )
+      ).data.price_preference
+
+      pricePreference2 = (
+        await api.post(
+          "/admin/price-preferences",
+          {
+            attribute: "currency_code",
+            value: "EUR",
+            is_tax_inclusive: true,
+          },
+          adminHeaders
+        )
+      ).data.price_preference
+    })
+
+    describe("/admin/price-preferences", () => {
+      describe("POST /admin/price-preferences", () => {
+        it("creates a price preference", async () => {
+          const newPricePreference = (
+            await api.post(
+              "/admin/price-preferences",
+              {
+                attribute: "region_id",
+                value: "region-2",
+                is_tax_inclusive: true,
+              },
+              adminHeaders
+            )
+          ).data.price_preference
+
+          expect(newPricePreference).toEqual(
+            expect.objectContaining({
+              attribute: "region_id",
+              value: "region-2",
+              is_tax_inclusive: true,
+            })
+          )
+        })
+
+        it("creates a price preference with false tax inclusivity by default", async () => {
+          const newPricePreference = (
+            await api.post(
+              "/admin/price-preferences",
+              {
+                attribute: "region_id",
+                value: "region-2",
+              },
+              adminHeaders
+            )
+          ).data.price_preference
+
+          expect(newPricePreference).toEqual(
+            expect.objectContaining({
+              attribute: "region_id",
+              value: "region-2",
+              is_tax_inclusive: false,
+            })
+          )
+        })
+      })
+
+      describe("GET /admin/price-preferences", () => {
+        it("returns a list of price preferences", async () => {
+          const response = (
+            await api.get("/admin/price-preferences", adminHeaders)
+          ).data.price_preferences
+
+          expect(response).toEqual(
+            expect.arrayContaining([
+              expect.objectContaining({
+                attribute: "region_id",
+                value: "region-1",
+                is_tax_inclusive: true,
+              }),
+              expect.objectContaining({
+                attribute: "currency_code",
+                value: "EUR",
+                is_tax_inclusive: true,
+              }),
+            ])
+          )
+        })
+        it("filters price preferences by attribute", async () => {
+          const response = (
+            await api.get(
+              "/admin/price-preferences?attribute=region_id",
+              adminHeaders
+            )
+          ).data.price_preferences
+
+          expect(response).toEqual(
+            expect.arrayContaining([
```

Comment:
nit: I think this shouldn't use `arrayContaining` as that could give us a false positive test result:

Reply:
Yes, copy-paste mistake. Nice catch 👍